saleor/graphql/core/mutations.py

```diff
@@ -272,8 +273,10 @@ def check_permissions(cls, context, permissions=None):
         if context.user.has_perms(permissions):
             return True
         service_account = getattr(context, "service_account", None)
-        if service_account and service_account.has_perms(permissions):
-            return True
+        if service_account:
+            if AccountPermissions.MANAGE_STAFF in permissions:
```

Comment:
I would add some comment here explaining that for now MANAGE_STAFF is not supported for service accounts.

Reply:
Documenting it as well? Before we get bug tickets about it

Reply:
Yup we will take care of that in saleor-docs.